runtime/jcl/common/jclcinit.c

```diff
@@ -66,127 +66,128 @@ jint computeFullVersionString(J9JavaVM* vm)
 {
 	VMI_ACCESS_FROM_JAVAVM((JavaVM*)vm);
 	PORT_ACCESS_FROM_JAVAVM(vm);
-	const char *osarch;
-	const char *osname;
-	const char *vmVersion = NULL;
-	const char *gcVersion = NULL;
-	char fullversion[512];
-	char vminfo[512];
-#ifdef J9VM_INTERP_NATIVE_SUPPORT
-	J9JITConfig *jitConfig;
-#endif
+	const char *osarch = NULL;
+	const char *osname = NULL;
+	const char *j2se_version_info = NULL;
+	const char *jitEnabled = "";
+	const char *aotEnabled = "";
+	const int bufferSize = 512;
+	char fullversion[bufferSize];
+	char vminfo[bufferSize];
+
+
 #if defined(J9VM_INTERP_NATIVE_SUPPORT)
-	UDATA jitEnabled = 0;
-	UDATA aotEnabled = 0;
-#endif
+	J9JITConfig *jitConfig = vm->jitConfig;
+
+	if (NULL != jitConfig) {
+		if (jitConfig->runtimeFlags & J9JIT_JIT_ATTACHED) {
```

Comment:
This condition should be written as `J9_ARE_ALL_BITS_SET(jitConfig->runtimeFlags, J9JIT_JIT_ATTACHED)`.
The test for `J9JIT_AOT_ATTACHED` below should be adjusted similarly.